.github/actions/verify-submodule-pull-request/Check-Parent.ps1

```diff
@@ -4,7 +4,7 @@
 )
 
 $url = "https://api.github.com/repos/$Repository/pulls?state=open&per_page=100"
-$titles = curl -s -H 'Accept: application/vnd.github.v3+json' $url | ConvertFrom-Json | ForEach-Object { $PSItem.title }
+$titles = Invoke-WebRequest -s -H 'Accept: application/vnd.github.v3+json' $url | ConvertFrom-Json | ForEach-Object { $PSItem.title }
```

Comment:
It is not. None of the parameters are interoperable.

Reply:
Just using curl here is fine BTW, `Invoke-WebRequest` isn't better.

Reply:
Invoke-Analyzers with the Fix switch replaced this, because curl is considered an alias, so Invoke-WebRequest would be better, because we can expect that it does the same thing (AFAIK because the curl available on Windows is not compiled from the same code as what's available on Linux, but pwsh is the same cross-platform code). We also used Invoke-WebRequest already in other repos/scripts instead of curl.

Reply:
Let's see, but please make sure to test this script's action/workflow too.

Reply:
Will do.

Reply:
These actions/workflows were executing according to the issue branch in the last run, still all green.